### ISSUE TITLE
[Feature request] Disable SDL Controller in Options

### ISSUE THREAD

**Issue by Erick8x** [exelix11/SysDVR#315]
**Is your feature request related to a problem? Please describe.**

When i try to stream my switch and another person are playing with controller on the same PC, the screen prompts every time a message to close the client. This problem doesn't occours in 6.0.

**Describe the solution you'd like**

Have a configuration to disable any input from controllers.

**Describe alternatives you've considered**

Tried to use a bunch of XInput/Dinput blockers, but any worked.

**Comment by LCLitaiff** [comment]
Quite annoying problem! Please, just add an option to disable this feature or at least configure it.

**Comment by exelix11** [comment]
Good point didn't think of that, i'll add it to the backlog for the next update.
As a workaround you can use the command line interface with the legacy player in cmd do `sydvr-client.exe bridge 192.168.1.5 --legacy` (replace 192.... with your console ip) for network mode and `sysdvr-client.exe usb --legacy` for usb.

The legacy player is the old one with no GUI.
